{{ cookiecutter.project_slug }}/Makefile

```diff
@@ -4,6 +4,8 @@
 #
 ###############################################
 
+.PHONY: requirements
```

Comment:
Why do you need this if you have it below?

Reply:
It conflicts with the requirements folder. @felipemontoya https://www.gnu.org/software/make/manual/make.html#Phony-Targets